README.Rmd

```diff
@@ -1,68 +1,19 @@
----
-output:
-  - github_document
-  - md_document
----
-
-<!-- README.md is generated from README.Rmd. Please edit that file. -->
-
-```{r, echo = FALSE}
-knitr::opts_chunk$set(
-  collapse = TRUE,
-  comment = "#>",
-  fig.path = "man/figures/README-"
-)
-```
-
-# diseasy <a href="https://ssi-dk.github.io/diseasy/"><img src="man/figures/logo.png" align="right" height="138" alt="diseasy website" /></a>
-
-<!-- badges: start -->
-
-[![CRAN status](https://www.r-pkg.org/badges/version/diseasy)](https://CRAN.R-project.org/package=diseasy)
-[![CI tests](https://github.com/ssi-dk/diseasy/actions/workflows/all-workflows.yaml/badge.svg)](https://github.com/ssi-dk/diseasy/actions/workflows/all-workflows.yaml)
-[![codecov](https://codecov.io/gh/ssi-dk/diseasy/branch/main/graph/badge.svg)](https://app.codecov.io/gh/ssi-dk/diseasy)
-
-<!-- badges: end -->
+{% extends '.github/templates/README_template.Rmd' %}
 
+{% block body %}
 ## Overview
-The `diseasystore` package provides feature stores implemented in R
-specifically designed for serve disease data for epidemic preparedness.
-
-What makes a `diseasystore` special, is that features can be
-automatically coupled and stratified within the `diseasystore` package.
-Consult the Quick start vignette to see it in action
-(`vignette("quick-start", package = "diseasystore")`).
-
-The package forms the data-backbone of the `{diseasy}` package.
-
-## Handling of diverse data sources
-Different data sources are handled by individual `diseasystores` which
-each facilitate access to the relevant disease data for the given data
-source.
-
-Data for different diseases will typically be structured in different
-ways. The `diseasystore` package currently implements the Google Health
-COVID-19 Open Repository with more one `diseasystores` the way.
-
-The `diseasystore` package is designed to handled both individual-level
-data (examples to come) and semi-aggregated (typically publicly
-available) data.
-
-If the data is at the individual-level, the feature store is fully
-dynamic and can adapt to (virtually) any stratification that the user
-specifies. If the data conversely is semi-aggregated, the data can only
-be stratified at the levels of the semi-aggregation (or at higher
-levels).
-
-## Installation
-```{r, eval = FALSE}
-# Install diseasy from CRAN:
-install.packages("diseasy")
-
-# Alternatively, install the development version from github:
-# install.packages("devtools")
-devtools::install_github("ssi-dk/diseasy")
-```
-
-## Usage
-For basic usage examples, see `vignette("quick-start")`.
+Leveraging combinatorial building for ensemble forecasting in disease
+modelling.
+
+## The power of combinations
+`diseasy` is framework for disease modelling that is built around the
+philosophy of combinatorial building. Even from a small number of
+modules, an ensemble consisting of a vast array of disease models can be
```

Comment:
I've notice that elsewhere you have started talking about templates instead of modules, should this apply here too?